build/tools/esp32/esptool_py/Makefile

```diff
@@ -74,7 +74,7 @@ $(APP_BIN_UNSIGNED): $(APP_ELF) $(ESPTOOLPY_SRC)
 
 ESPTOOL_ALL_FLASH_ARGS += $(ESPTOOL_BOOTLOADER_FLASH_ARGS)
 
-ALL:flash
+ALL:flash romfs
 
```

Comment:
it looks fine. it keeps the function "make download ALL" same as other arch, like (artik).
and also if no romfs image for current configuration, it does not take effect.